scrapers_next/ar/people.py

```diff
@@ -131,3 +131,235 @@ def process_item(self, item):
+    selector = XPath("//div[contains(@class,'col-sm-6')]")
+    chamber = "upper"
+
+    def process_item(self, item):
+        description = item.text_content()
+        nameFromDescription = description.split("\r\n")[2]
+        link = list(item.iterlinks())
+        (element, attr, url, position) = link[1]
+        p = PartialMember(
+            name=nameFromDescription, chamber="upper", url=self.source.url
+        )
+        return SenDetail(p, source=url)
+
+
+class AssemblyDetail(HtmlPage):
+    example_source = "https://www.arkansashouse.org/district/1"
+
+    def process_page(self):
+        heading = CSS(".py-sm-6").match(self.root)[0]
+        name = self.input.name
+        line2 = heading[2].text_content().split(" ")
+        party = line2[0]
+        districtName = line2[2]
+        districtNumber = ""
+        for c in districtName:
+            if c.isdigit():
+                districtNumber += c
+
+        leftInfobox = CSS(".col-lg-3").match(self.root)[0]
+
+        table = {
+            "Seniority": "",
+            "Occupation": "",
+            "Religion": "",
+            "Past Service": "",
+        }
+
+        for k in table.keys():
+            value = XPath("//dt[. = '" + k + "']//following-sibling::dd").match(
+                leftInfobox
+            )[0]
+            valueString = value.text_content()
+            if len(valueString) > 0:
+                table[k] = valueString
+
+        table["Name"] = name
+        table["District"] = districtNumber
+        table["Title"] = "Representative"
+        table["Party"] = party
+
+        image = XPath("//div[contains(@class,'col-lg-3')]//a/@href").match(self.root)[0]
+
+        emailContainer = XPath("//p[contains(@class,'mb-0')]/a").match(self.root)[0]
+        table["Email"] = emailContainer.text_content()
+
+        officeContainer = XPath("//p[contains(@class,'mb-0')]").match(self.root)[1]
+        table["Phone"] = officeContainer[2].text_content()
+        addressArray = XPath("//p[contains(@class,'mb-0')]/text()").match(self.root)
+
+        address = ""
+        started = False
+        for c in addressArray[3]:
+            if c.isalnum():
+                started = True
+            if started:
+                address += c
+        address += " "
+        address += addressArray[4]
+        table["District Address"] = address
+
+        # Get the district from the URL as a fallback
+        district = table["District"]
+        if not "District" in table or len(district) < 1:
+            urlParts = self.source.url.split()
+            district = urlParts[len(urlParts) - 1]
+
+        p = ScrapePerson(
+            name=name,
+            state="ar",
+            chamber=self.input.chamber,
+            party=table["Party"],
+            image=image,
+            district=district,
+            email=table["Email"],
+        )
+        p.add_source(self.source.url)
+        p.add_source(self.input.url)
+
+        for key in table:
+            if (
```

Comment:
`if key in ["Phone", "Name", "Email", "District", "Biography", "District Address", "Phone"]:`?